config/routes.rb

```diff
@@ -72,6 +72,11 @@
       resources :events
     end
 
+    get "/subgroups/new", to: 'groups#new_subgroup'
```

Comment:
as a general matter, i'd want to handle the subgroup route as a nested resource like:

```
resources :groups do 
  resources: :subgroups
end
```

as with: https://github.com/affinityworks/main/blob/e1f6391e39ca61521253a7ed3ce05b1bf893ecc7/config/routes.rb#L66

which will automatically route `groups/:id/subgroups/new` to `subgroups#new`. (and give you the `group_id` and subgroup `id` params for free.

curious if there's a reason why you opted to put it on the groups controller here?